gulp/config.js

```diff
@@ -96,8 +96,8 @@ module.exports = {
             dest: `${dest}/docs/css/`
         },
         includes: {
-            src: `./node_modules/@mozilla-protocol/*/*.scss`,
```

Comment:
This selector meant we we're only looking one level deep :/

Reply:
what a difference an asterisk makes...